modules/Bio/EnsEMBL/Analysis/Hive/Config/production_rnaseq_db.pm

```diff
@@ -591,17 +592,40 @@ sub pipeline_analyses {
        cmd => "sudo -u genebuild find " .catdir('#production_ftp_dir#', 'species', ucfirst($self->o('species_name'))). " -user genebuild -exec chmod g+w {} \\;"},
      -rc_name    => '2GB',
      -flow_into => {
-	1 => ['delete_data_files'],
+        1 => ['check_main'],
      },
    },
 
+   {
+      -logic_name => 'check_main',
+      -module => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+      -parameters => {
+         cmd => 'if [[ $(#search_query#) ]]; then exit 0; else exit 42;fi',
+         return_codes_2_branches => {'42' => 2},
+         search_query => 'grep -w '. "'".$self->o('species_name')."' ".$self->o('species_list') . ' | cut -f 2',
+
+      },
+    -rc_name => '2GB',
+    -flow_into => {
+         1 => ['copy_to_main_ftp'],
+         2 => ['delete_data_files'],
+     }
+   },
+      
    { 
      -logic_name => 'delete_data_files',
      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
      -parameters => { 
         cmd => 'rm -r '. $self->o('merge_dir') . '/* ' .$self->o('output_dir') . '/* ',
      },
-    },
+   },
+
+   { #for now we won't do anything with the file, need to get SOP for moving to the main FTP space
```

Comment:
How this analysis is executed by eHive? Is it always passed on?

Reply:
It will just pass because there is nothing to do. 
The Dummy module is used as way to structure the pipeline, like the 'fan' analyses in the main pipeline, in those cases it doesn't have anything to do but it does have information about how to move on to the next analyses via the 'flow_into' parameter. In this case it doesn't do anything and doesn't have anything to flow to... it's just a placeholder for the actual analyses that will do the file moving.